src/utils/gaa-test.js

```diff
@@ -1798,6 +1826,10 @@ describes.realWin('GaaGoogle3pSignInButton', {}, () => {
     sandbox.stub(self.console, 'warn');
 
     sandbox.stub(self, 'open');
+    const elements = self.document.getElementsByTagName('div');
```

Comment:
maybe leave a comment on this setup task? something like
```
// Makes sure no div elements on the test dom exist since this test creates some
```

Reply:
sg